lib/publiccloud/azure.pm

```diff
@@ -158,8 +158,8 @@ sub upload_img {
     $blobs =~ s/^\s+|\s+$//g;    # trim
     my @blobs = split(/\n/, $blobs);
     if (grep(/$img_name/, @blobs)) {
-        die("The upload_img() subroutine has been called even tho the $img_name blob exists. " .
-              "This means that ccloud/upload_image test module did not properly detected it.");
+        record_soft_fail('BLOB EXISTS', "The upload_img() subroutine has been called even tho the $img_name blob exists. " .
```

Comment:
SAME AS ABOVE!!1!

```suggestion
        record_soft_fail('Image exists', "upload_img() called even though the $img_name blob exists.\nThis means that publiccloud/upload_image test module did not properly detect it.");
```

Reply:
Same as above + it's not image but blob. See line `157`: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/9a13199b5fc32abdcca93661a7a272e33a374dd7/lib/publiccloud/azure.pm#L157

Reply:
Ok